package raft;
option cc_generic_services = true;

message IndexRequest {};
message IndexResponse {};

service raft_stat {
    rpc default_method(IndexRequest) returns (IndexResponse);
}

